Macedonian/Majka.mdown

```diff
@@ -0,0 +1,5 @@
+Денес добив повик од мајка ми кога бев на работа. Ми рече дека останува
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.